CHANGELOG.md

```diff
@@ -2,11 +2,13 @@
 
 ## Master (Unreleased)
 
-- Fix an infinite loop error when `RSpec/ExcessiveDocstringSpacing` finds a description with non-ASCII leading/trailing whitespace. ([@bcgraham])
-- Add new `RSpec/Eq` cop. ([@ydah])
-- Fix an incorrect autocorrect for `RSpec/ReceiveMessages` when return values declared between stubs. ([@marocchino])
+## 2.24.0 (2023-09-07)
+
```

Comment:
Ah I see a typo in the change below: the two new cups instead of cops

Reply:
Thanks. Fixed.